src/main/java/com/parth/StudentManagementMyBatisJwt/StudentManagementMyBatisJwtApplication.java

```diff
@@ -1,11 +1,19 @@
 package com.parth.StudentManagementMyBatisJwt;
 
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
 import org.mybatis.spring.annotation.MapperScan;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-@MapperScan("com.parth.StudentManagementMyBatisJwt.repository")
+@MapperScan({"com.parth.StudentManagementMyBatisJwt.repository","com.parth.StudentManagementMyBatisJwt.repository.jwtUser"})
+@SecurityScheme(
```

Comment:
For this project/use case, I think we don't need Security Scheme annotation to be added everywhere. You can add it in a central place in the openAI config class.

Reply:
Done